config/routes.rb

```diff
@@ -86,6 +86,7 @@
       get :completed_simulations_count
       get :experiment_stats
       get :experiment_moes
+      get :experiment_results
```

Comment:
Nawiązując do poprzedniego komentarza, skoro to jest i tak w
resources :experiments do
    member do
        ...
to czy jest tutaj potrzeba a nawet w ogóle sens, żeby te motody nazywały się experiment_... ?